packages/muffincss-postcss/src/tests/resets/output.test.css

```
@@ -0,0 +1,453 @@
+  }
+}
+@layer reset-def {
+  *,
+  ::after,
+  ::before,
+  ::backdrop,
+  ::file-selector-button {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+    border: 0solid;
```

Biome (2.1.2)

[error] Unexpected unknown unit: solid

See MDN web docs for more details.
Use a known unit instead, such as:


(lint/correctness/noUnknownUnit)